packages/ExternalLink/src/ExternalLink.js

```diff
@@ -32,7 +32,7 @@ function ExternalLink(props) {
 
   return (
     <sc.ExternalLink
-      aria-label={ariaText || children}
+      aria-label={ariaText || null}
```

Comment:
This would not have worked if `children` was a node instead of a string.  But it's also unnecessary, since the content of the `<a>` will be used as the accessible name by AT automatically if there's no `aria-label`.

Reply:
no for this, but the way I did it on the DataGrid its that if the user passes it down a string, its use automatically as a a11ytext, otherwise if it's a node or function I fire a `console.warn` asking for the developer to provide an `allyLabel` prop to describe the content on the node.

Reply:
Yeah, I think that's a good pattern, and I've seen it in the Takeover too:
https://github.com/acl-services/paprika/blob/master/packages/Takeover/src/Takeover.js#L75-L85

Reply:
Maybe this would be useful to have in the `helpers` package?